modules/ROOT/pages/setup-payload-storage-API.adoc

```diff
@@ -34,25 +38,18 @@ https://www.mulesoft.com/exchange/com.mulesoft.b2b/partner-manager-content-stora
 You’ll need this information when you associate Anypoint Partner Manager with your payload storage API.
 Your payload storage API must be accessible over the internet via HTTPS. Ensure that the appropriate authentication mechanisms are implemented to enable access over HTTPS.
 
-== Configure the Payload Storage API
+== Add the Payload Storage API
 
-. In Anypoint Platform, open Anypoint Partner Manager. 
-. In the navigation menu on the left, select your organization’s profile (host) page.
-. In the *Payload storage* section of the page, click *Edit* to add the path to your payload storage.  
+. In Anypoint Platform, open Anypoint Partner Manager.
+. In the navigation menu on the left, select your organization’s host page.
```

Comment:
Note that previous tasks start with "In the Partner Manager left navigation menu,..."
We should standardize how all the tasks start, deciding whether we need to have the audience actually navigate to and open Partner Manager. 
Personally, I would start each task with "From the navigation menu [on the left], select...."

Reply:
Done!